### ISSUE TITLE
[Idea] Slide Canceller

### ISSUE THREAD

**Issue by WeMayNeverKnow** [YTVanced/Vanced#972]
**Description**
When accidentally sliding the seek while watching a video there will be a cancel pop up to cancel the seek and go back to the original time (before siding)

The button is like in moba games where you can cancel your skill.

This would be very helpful for people who watches long videos and accidentally did something because it's hard to go back to past

I feel like my explanation is very bad sorry

**Comment by DraconicNEO** [comment]
That's a good idea, although isn't the video interface getting a bit crowded with the auto-play, cast, captions, and recently sponsor block voting buttons. Don't get me wrong that is a good idea oh, but we don't want to clutter up the UI too much.

**Comment by WeMayNeverKnow** [comment]
It will only pop up when the button is hit I guess?

**Comment by TrianguloY** [comment]
A button will force you to use another finger. I think it's better to simply cancel seeking when you move the finger away from the bar (when the vertical distance is greater than twice the bar height for example).
This way if you started seeking by mistake, or you simply want to cancel, just move the finger away from it so the red dot will move back to the current position, and then you can release the finger safely.

**Comment by WeMayNeverKnow** [comment]
That's what I was actually saying just like those on games you just have to move it to that place ☺️

**Comment by TheBinaryGuy** [comment]
@TrianguloY A good visual indicator would be if you start moving your finger away from seek bar, an X / close button would popup at the bottom of the screen and if you move your finger to that and release then that would cancel seeking.

**Comment by TrianguloY** [comment]
> @TrianguloY To add to your suggestion, a good visual indicator would be if you start moving your finger away from seek bar, an X / close button would popup at the bottom of the screen and if you move your finger to that and release then that would cancel seeking.

That was not my suggestion but @WeMayNeverKnow 

My suggestion was to cancel seeking when moving and releasing the finger away from the bar, without visual indicator (only the red dot going back to the original position). They are practically the same idea, but one requires ui changes and the other doesn't

**Comment by TheBinaryGuy** [comment]
@TrianguloY Yeah I know, I meant your suggestion of moving off of sidebar w/o any inidcator.

People would probably need some sort of visual indicator I guess, otherwise they might get confused on why seeking isn't working if they like to seek from middle or bottom of the screen. I myself start seeking a video and move my fingers vertically to the middle of my screen (especially if it's a long video) to see the preview and time properly and I can get more precise that way as well (or at least I feel more precise, maybe it's just a placebo).

**Comment by TrianguloY** [comment]
That's a good point. 
Maybe another alternative instead of a button would be to place a fading to black at the top/bottom of the screen, with an x in the middle if possible. Sort of like when you move the pip window, so that you know if you move the finger to that side and continue moving to the end of the screen, the seeking is cancelled.

**Comment by TheBinaryGuy** [comment]
Yep that sounds good, that would solve the issue of having too many UI elements on the screen.

**Comment by WeMayNeverKnow** [comment]
What if we also add move to previous area?
Because sometimes you just accidentally remove your hand 😂

**Comment by m0nk3ybraindead** [comment]
Updates on this?
